src/backgroundScript.js

```diff
@@ -30,16 +30,15 @@ const backgroundScript = (selectors, colors) => {
   }
   roots.forEach((root) => {
     selectors.forEach((selector) => {
-      try {
-        const randomColor = colors[Math.floor(Math.random() * colors.length)];
-        root.querySelectorAll(selector).forEach((element) => {
+      const randomColor = colors[Math.floor(Math.random() * colors.length)];
+      const matchedElements = root.querySelectorAll(selector);
+      if (matchedElements && matchedElements.length > 0) {
+        matchedElements.forEach((element) => {
           injectElementStyle(element, randomColor);
           injectSpanElement(element, randomColor, selector);
-
-          count++;
```

Comment:
are you sure you are not removing a feature that is already on main?

Reply:
My bad, mistake done when resolving conflicts. Will fix that.